tests/jax/components/training/sample.py

```diff
@@ -0,0 +1,445 @@
+from types import SimpleNamespace
```

Comment:
Please update the name to be `step_test_sample.py` and add the licence to the file :)